### ISSUE TITLE
Make the user docs for the valve

### ISSUE THREAD

**Issue by aott33** [Open-Mobile-Robotics/open-process-library#7]
I think the documentation for DMC's Siemens Open Library is a good reference for documenting the library. I found their documentation very helpful when first starting out using their library. Example below:

[4- Siemens Open Library - Detailed Block Overview.pdf](https://github.com/Open-Mobile-Robotics/open-process-library/files/15427810/4-.Siemens.Open.Library.-.Detailed.Block.Overview.pdf)

**Comment by zantiu** [comment]
Yes. The Rockwell PlantPax object library docs are also a good example.

**Comment by russwaddell** [comment]
> ...Example below:
> 
> [4- Siemens Open Library - Detailed Block Overview.pdf](https://github.com/Open-Mobile-Robotics/open-process-library/files/15427810/4-.Siemens.Open.Library.-.Detailed.Block.Overview.pdf)

See Section 7 starting p. 50 for valve tags and diagrams

**Comment by russwaddell** [comment]
> Yes. The Rockwell PlantPax object library docs are also a good example.

https://literature.rockwellautomation.com/idc/groups/literature/documents/rm/proces-rm001_-en-p.pdf

See valve examples on p. 34-35. Not sure if this is the documentation you had in mind for PlantPax

**Comment by zantiu** [comment]
No Rockwell/PlantPax has dedicated docs for the objects.
This is an older version example of the valve:
https://literature.rockwellautomation.com/idc/groups/literature/documents/rm/syslib-rm014_-en-e.pdf
But I can't find the latest one.

It's quite extensive. Probably a bit too much.
